src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/edit.html.twig

```diff
@@ -70,3 +70,7 @@
     {% include 'KunstmaanNodeBundle:NodeAdmin:_modals.html.twig' %}
 
 {% endblock %}
+
+{% block extracss %}
+    <link rel="stylesheet" href="/frontend/css/admin-style.min.css">
```

Comment:
We need to check if the file exists, because when you update a site without admin styles, the file will not exist. Also use asset() function.